src/client/features/search/query-entity-info.js

```diff
@@ -154,6 +134,56 @@ const getUniprotInfo = ids => {
   });
 };
 
+const getHgncInfo = hgncSymbols => {
+  const hgncSymbol = Object.keys(hgncSymbols)[0];
+
+  return ServerAPI.getHgncInformation( hgncSymbol ).then( result => {
+    const geneResults = result.response.docs;
+    return geneResults.map(gene => {
```

Comment:
https://www.genenames.org/help/rest-web-service-help
>Fetch will not accept wildcards or multiple queries 

HGNC fetch can only accept one token and return one result, so response.dcos is always = 1. Plus `getHgncInformation()` is only called when the token is prefixed (implying there is only one token).

Maybe for consistence reason, I should wrap the `fetch` within a loop so that it accept multiple tokens like `getUniprotInfo()` and `getHgncInfo()`.

Reply:
Promises like `fetch()` should not go in a loop.  Use `Promise.all( someList.map(doSomeFetch) )` or similar